src/pages/Popup/index.tsx

```diff
@@ -9,32 +9,70 @@ import { ErrorMessage, LoadingMessage } from '../../components/PageMessages';
 import { ContentDecoder } from '../../components/ContentDecoder';
 
 import { fetchAnalyze, fetchDescription } from '../../shared/api';
+import { logPageView, logSearchClick } from '../../shared/logs';
 import * as Styled from './index.styled';
 
 import '../../shared/reset.css';
 import '../../shared/font.css';
 
-import { logPageView, logSearchClick } from '../../shared/logs';
+const chatError =
+    "GPT-3's experiencing some technical difficulties, but don't worry, our team's on it. In the meantime, give it another try or holla at us if you need a hand.";
 
 function Panel() {
     const [to, setTo] = useState('');
     const [chainId, setChainId] = useState('0x1');
     const descriptionResult = useAsyncCallback(async (chainId, to) => fetchDescription(chainId, to));
     const analyzeResult = useAsyncCallback(async (chainId, to) => fetchAnalyze(chainId, to));
 
+    function saveSessionState() {
```

Comment:
With the new server arch we discussed we'l. be able to easily store loading states as well, this will make this feature much better